docs/google/floquet.ipynb

```diff
@@ -0,0 +1,1348 @@
+{
```

Comment:
<p>I think this should be completely removed. From what I saw on the other notebooks in this repository, the assumption is that you start from python environment on current head.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/quantumlib/Cirq/pull/3765/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Reply:
<p>The install is so it can run on colab in a clean environment, though. </p>

Reply:
@mpharrigan any thoughts on this?

Reply:
<p>Updated to install from cirq/master and verified it works in a clean environment. However, it does not work in colab because of https://github.com/quantumlib/Cirq/issues/3774, but I don't think that will be resolved for this issue. </p>

Reply:
<p>Removed the colab link (above) but kept the cirq/master install for now.</p>

Reply:
Hi guys, I'm adding a guide on notebooks: https://github.com/quantumlib/Cirq/pull/3781
Thus, I think we should keep the header - that's required for devsite, and we should have this as a pre-release notebook.

Reply:
Added the header back & changed to pip install cirq --pre. Also, added the pre-release disclaimer from #3781.